src/Marten.Testing/Bugs/Bug_PR_1412_spaces_in_table_name_cause_schema_patch_to_fail.cs

```diff
@@ -0,0 +1,143 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using Marten.Schema;
+using Marten.Storage;
+using Xunit;
+
+namespace Marten.Testing.Bugs
+{
+    public class Bug_PR_1412_spaces_in_table_name_cause_schema_patch_to_fail : IntegratedFixture
+    {
+
+        internal class testSchema : IFeatureSchema
+        {
+            public ISchemaObject[] Objects => SchemaObjects().ToArray();
+
+            public Type StorageType => GetType();
+
+            public string Identifier { get; set; }
+
+            public IEnumerable<Type> DependentTypes()
+            {
+                return new List<Type>();
+            }
+
+            public bool IsActive(StoreOptions options)
+            {
+                return true;
+            }
+
+            public void WritePermissions(DdlRules rules, StringWriter writer)
+            {
+                // No permissions!
+            }
+
+            protected virtual IEnumerable<ISchemaObject> SchemaObjects()
+            {
+                List<ISchemaObject> objects = new List<ISchemaObject>();
+                return objects;
+            }
+
+        }
+
+        internal class spaceAfterTableNameSchema: testSchema
+        {
+            protected override IEnumerable<ISchemaObject> SchemaObjects()
+            {
+                var table = new Table(new DbObjectName("test_space_after"));
+                table.AddColumn("space_after ", "int");
+                return new List<ISchemaObject>
+                {
+                        table
+                };
+            }
+        }
+        [Fact]
+        public void space_after_table_name_does_not_cause_exception()
+        {
+            StoreOptions(_ =>
+            {
+                _.AutoCreateSchemaObjects = AutoCreate.CreateOrUpdate;
+                _.Storage.Add(new spaceAfterTableNameSchema());
+            });
+            try
+            {
+                theStore.Schema.ApplyAllConfiguredChangesToDatabase();
+                theStore.Schema.ApplyAllConfiguredChangesToDatabase();
+            }
+            catch (Exception e)
```

Comment:
**FIX:** @malscent coud you use `Should.NotThrow` (see [sample](https://github.com/JasperFx/marten/blob/1128d6f6a63188087dd34645f735b015f76bf4c7/src/Marten.Testing/StoreOptionsTests.cs#L114)). Instead of the catch? Same apply for other places.

Reply:
@oskardudycz I could not find an example in your link.  Did you send the correct one?

Reply:
It should lead you to:
![obraz](https://user-images.githubusercontent.com/5562528/72729992-be2a6400-3b90-11ea-956a-e4c30c8d9d3e.png)

Reply:
Haha it did.. i'm just blind.  I'll get that corrected ASAP